docs/configuration.md

```diff
@@ -22,12 +25,19 @@ Log level of the controller.
 * NODE_ADDRESS_TYPE, *default:* "external"  
 Address type of the nodes. This might be set to internal, if your external IPs are  registered as internal IPs on the node objects (e.g. if you have no cloud controller manager). Can be "external" or "internal".
 
+* NODE_LABEL_SELECTOR
+Optionally restrict the searched nodes to assign floating ips to by a label selector.
+
 * NODE_NAME  
 Name of the scheduled node. Should be invoked via fieldRef to spec.nodeName
 
 * NAMESPACE  
 Namespace the pod is running in. Should be invoked via fieldRef to metadata.namespace
 
+* POD_LABEL_SELECTOR, *defailt:* hcloud-fipcontroller
+Labels selector to find deployment pods with. This should be the same value as used in the deployment.
+Can be ignored if no label was specified.
```

Comment:
> How are multiple labels configured?
With a label selector it is possible to select multiple labels.

I should have maybe written the text differently. 
With ignoring it I meant, that if you don't want to select the hetzner vms with pods but with nodes instead, you don't need to worry about the variable, because the fip-controller will automatically search the nodes, in case it doesn't find any pods.